cpu/msp430/cc2520-arch-sfd.c

```diff
@@ -44,6 +44,7 @@ ISR(TIMERB1, cc2520_timerb1_interrupt)
   ENERGEST_ON(ENERGEST_TYPE_IRQ);
   /* always read TBIV to clear IFG */
   tbiv = TBIV;
+  (void)tbiv;
```

Comment:
I suggest to either look for an alternative like defining `tbiv`as `volatile` or at least add a comment why this line was added.